lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -274,14 +274,15 @@ class SemanticsObject {
     // DOM nodes created for semantics objects are positioned absolutely using
     // transforms.
     element.style.position = 'absolute';
+    element.setAttribute('id', 'flt-semantic-node-$id');
```

Comment:
Probably not a big deal: Is this test only? if so, it can be put into assert

Reply:
Yeah, accessibility node count tends to be small enough for this extra info to not matter in terms of performance. We also aggressively reuse accessibility nodes, so this constructor is only called once for the entire lifetime of a node with an ID. It is useful for debugging production apps, so I'd like to keep it. If this becomes a performance problem, it's easy enough to remove. It's not used for anything.

Reply:
sg